xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -824,6 +824,11 @@ $value##
 
     #end
     #displaySearchForm()
+    #foreach ($extension in $services.uix.getExtensions("org.xwiki.platform.search.widget"))
```

Comment:
it seems you're adding a SOLR only extension point and not a generic search one, so I would use "solr" somewhere in the UIXP id. WDYT?